alicloud/service_alicloud_slb.go

```diff
@@ -152,7 +152,7 @@ func (s *SlbService) DescribeSlbBackendServer(id string) (*slb.DescribeLoadBalan
 }
 
 func (s *SlbService) DescribeSlbListener(id string, protocol Protocol) (listener map[string]interface{}, err error) {
```

Comment:
If the id has contained protocol, the method `DescribeSlbListener ` should only have one parameter `id`.